homeassistant/components/husqvarna_automower/__init__.py

```diff
@@ -53,6 +59,9 @@ async def async_setup_entry(hass: HomeAssistant, entry: AutomowerConfigEntry) ->
 
     coordinator = AutomowerDataUpdateCoordinator(hass, automower_api, entry)
     await coordinator.async_config_entry_first_refresh()
+    available_devices = list(coordinator.data)
+    if coordinator.config_entry:
```

Comment:
How's can the config entry of the coordinator ever be none? If its a typing issue preferable retype the field in the subclass or use an assert here.

Reply:
Yes, it's a typing issue. I changed it.